js/chatbot/shared/ScrollDown.svelte

```diff
@@ -0,0 +1,15 @@
+<svg
```

Comment:
I just made this SVG with LLM for this component, but plz tell me if there is already a usable resource for this purpose.
Also, should this be put in the `@gradio/icons` package?

Reply:
Yes, can you add it to the icons package.

Reply:
Fixed